packages/generator-react-server/test/app.js

```diff
@@ -24,6 +24,46 @@ test('generator-react-server:app creates default files', async t => {
 	t.false(await exists('docker-compose.yml', testDir));
 });
 
+test('generator-react-server:app .reactserverrc matches react-server-cli default option keys', async t => {
+	let testDir;
+	await helpers.run(path.join(__dirname, '../generators/app'))
+		.inTmpDir(dir => {
+			testDir = dir;
+		})
+		.withPrompts({name: 'foo', dockerCfg: false})
+		.toPromise();
+
+	// Reading files here instead of requiring because we don't understand export default here.
+	const defaultOptions = await readFile('defaultOptions.js', path.join(__dirname, '../../react-server-cli/src'));
```

Comment:
Seems easier to just `require(path.resolve(...))` here... what's the benefit of reading and parsing like this?